plugins/woocommerce/client/legacy/css/twenty-nineteen.scss

```diff
@@ -1034,6 +1073,40 @@ table.variations {
 	}
 }
 
+.woocommerce-shipping-totals {
+	.select2-container {
```

Comment:
For non-block themes (Twenty Nineteen and Twenty Seventeen), add style to the `select2` inputs so they get the same appearance as the text inputs.

**Before**
![image](https://github.com/user-attachments/assets/64879b38-6c63-476f-8282-f06bef59511d)

**After**
![image](https://github.com/user-attachments/assets/2e6d7f77-4b01-4cc8-a524-b134057f1055)